content/flutter-publishing/publishing-to-google-play.md

```diff
@@ -85,6 +85,5 @@ Once you make all the preparations as described [above](../publishing/publishing
 11. Select **Enable Google Play publishing** at the top of the section to enable publishing.
 
 {{<notebox>}}
-**Note:** You can use the "**Wear OS Only**" track to manage Wear OS releases in Play Console. To target "**Wear OS Only**" track add a **wear:** prefix in the track name.
-e.g. **wear:internal**
+**Note:** To target "**Wear OS Only**" track, add **wear:** in the track name.
```

Comment:
do not delete the beginning of the sentence: **Note:** You can use the "**Wear OS Only**" track to manage Wear OS releases in Play Console. To target "**Wear OS Only**" track, add **wear:** in the track name